src/WebCompiler/Compile/LessCompiler.cs

```diff
@@ -121,9 +121,6 @@ private static string ConstructArguments(Config config)
             if (!options.IECompat)
                 arguments += " --no-ie-compat";
 
-            if (options.StrictMath)
```

Comment:
The first entry is at line 118 (hidden in the diff), either expand above or checkout here:
https://github.com/madskristensen/WebCompiler/blob/master/src/WebCompiler/Compile/LessCompiler.cs#L118